### DISCUSSION TITLE
Σ2 Grain Boundaries Tungsten Carbide

### DISCUSSION THREAD

**Discussion by StephanGen** [mtex-toolbox/mtex#781] [Ask Anything]
Hello,

I want to calculate the CSL 2 grain boundaries at WC/WC boundaries in cemented carbides and got the following error. Other CSL boundaries are calculated correctly.

Best wishes,

Stephan 

`>> gB2 = gB_WCWC(angle(gB_WCWC.misorientation,CSL(2,cS)) < 5*degree);
Error using  == 
Matrix dimensions must agree.

Error in orientation/dot (line 100)
  (~isa(o1,'rotation') || ~isa(o2,'rotation') || all(o1.i(:) == o2.i(:))));

Error in orientation/angle (line 17)
  omega = real(2*acos(abs(dot(o1,varargin{:}))));`

**Comment by kilir** [comment]
Hi,

I think Σ would always be odd, even Σ would be the same lattice points.

Cheers,
Rüdiger

**Reply by StephanGen** [reply]
Hi Rüdiger,

yes WC its special because its c/a ratio of around 1. Therefore it can get these special grain boundaries. I already tried to find a work around by compare the misorientation of the neighboured WC grains. Unfortnuately I struggled today to find correct conditions for that. There exists two types of Σ2 boundaries. A twist Σ2 grain boundary (two prism planes- crystal rotated 90° along [10-10]) and a tilt Σ2 grain boundary which is consist of a basal and a prismatic plane with the notation (0001)|(1-210) - I attached a picture, not the best visualisation but the best I found quickly. I think there is no way to distingish between both types by EBSD because they have the same orientation relationship. I tried the orientation.map function but the documentations says its only for directions.

Literature reference: https://ui.adsabs.harvard.edu/abs/2018AcMat.155..372P/abstract

![image](https://user-images.githubusercontent.com/11254604/108086401-9b5dce00-7076-11eb-816c-b14c7857f606.png)

**Reply by kilir** [reply]
Hi,
if I see this correctly, the misorientation for both relations is
```
cs = crystalSymmetry('-6m2')
mori = orientation.byAxisAngle(Miller(1,0,-1,0,cs),90*degree,cs,cs);
```
The decision whether you have a tilt or a twist boundary from 2D EBSD isn't straight forward unless you can use additional information, e.g. if the grain boundary trace corresponds always to one or a set of possible crystallographic planes; you can exclude twist boundaries if the rotation axis is parallel to the boundary trace etc. ..

Hope this helps.

Cheers,
Rüdiger

**Reply by StephanGen** [reply]
Thank you, it works with your code.  I will skip the declustering of the Σ2 grain boundaries in the first step because its currently more an option for my research.
